README.md

```diff
@@ -44,13 +44,19 @@ The following files and folders enable our continuous integration system.
    ```sh
    rosdep install --from-path src --ignore-src -y
    ```
+
+	- Some packages may have to be installed from source. For example, if realsesnse2_camera cannot be found, then search the error and find the ROS wiki page http://wiki.ros.org/realsense2_camera. Find the link to the source git repo and clone that into catkin_ws/src.
```

Comment:
Since we know the two repos that need this step and this is an exception to the normal rosdep workflow, can we just add instructions that say something like "clone ____ and ___ to src" like we've done with Keras.

Reply:
Fair